sdk/monitor/Azure.Monitor.Query/src/Azure.Monitor.Query.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>A library for querying Azure Monitor's Logs and Metrics data sources.</Description>
     <AssemblyTitle>Azure Monitor Query client library</AssemblyTitle>
-    <Version>1.1.0-beta.1</Version>
+    <Version>1.0.2</Version>
```

Comment:
I think we should bump the minor and release `1.1.0` since we're adding new public APIs per https://azure.github.io/azure-sdk/policies_releases.html#package-versioning.  .NET doesn't _really_ follow SemVer, but we've been pretty consistent on our team about only using patch releases for minor bug fixes.